src/cv/yolo_video_detect.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import imutils
+import time
+import cv2
+import os
```

Comment:
I recommend following this import statement ordering: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement

Reply:
organized the imports